test/__init__.py

```diff
@@ -128,7 +129,8 @@ def test_repository_files_exists(self):
                 self.assertTrue(
                     any(os.path.exists(f) for f in file_path),
                     msg=self.file_not_found_message.format(
-                        '" or "'.join(file_path)),
+                        '" or "'.join(file_path)
+                    ),
```

Comment:
Is the trailing comma really necessary?

Reply:
No, but in general, if another parameter was added (in practice, not likely with `assertTrue`), it would mean this previous line wouldn't be included in the diff. Happy to remove it if needed.

This is something [Black](https://github.com/ambv/black) could sort out automatically :)

Reply:
As I said before, using black would get rid of a bunch of overlapping PRs in sendgrid's repos.